indexing-service/src/main/java/org/apache/druid/indexing/worker/WorkerTaskManagerMonitor.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.indexing.worker;
+
+import com.google.inject.Inject;
+import org.apache.druid.indexing.common.task.Task;
+import org.apache.druid.java.util.emitter.service.ServiceEmitter;
+import org.apache.druid.java.util.emitter.service.ServiceMetricEvent;
+import org.apache.druid.java.util.metrics.AbstractMonitor;
+import org.apache.druid.query.DruidMetrics;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+public class WorkerTaskManagerMonitor extends AbstractMonitor
```

Comment:
Can we reuse WorkerTaskCountStatsMonitor itself?

Reply:
Wanted to emit this metric from indexer pods
We have seen frequent missing metrics issues when we enable more metrics which emit from the overlord

Reply:
Are those stats not available in indexer? You could selectively emit stats that are available on indexers but use the same monitor.

Reply:
+1 on using the existing `WorkerTaskCountStatsMonitor` as the new metrics fall in the same bucket.